src/routes/boot.ts

```diff
@@ -510,6 +510,7 @@ const loggedInBoot = async ({
         language: user.language || undefined,
         image: mapCloudinaryUrl(user.image),
         cover: mapCloudinaryUrl(user.cover),
+        defaultFeedId: isPlus ? user.defaultFeedId : null,
```

Comment:
Should we return the my-feed id (userId) or basically wouldn't really matter?

Reply:
null is default when you don't have anything set, frontend expects it as well